components/automate-ui/src/app/services/mfe-session/mfe-session.service.ts

```diff
@@ -0,0 +1,17 @@
+import { ActivatedRouteSnapshot, CanActivate, RouterStateSnapshot } from '@angular/router';
+
+export class MfeSessionService implements CanActivate {
+  canActivate(_route: ActivatedRouteSnapshot, state: RouterStateSnapshot): boolean {
+    let map_url = localStorage.getItem('url_mapping');
```

Comment:
```suggestion
    let map_url = localStorage.getItem('mfe_url_mapping');
```

Reply:
changes added in new commit [1e5dbbb](https://github.com/chef/automate/pull/8027/commits/1e5dbbbb09aeeb0798efbad56246b17a96264cd3)